spec/features/item_registration_csv_spec.rb

```diff
@@ -128,6 +132,32 @@
 
       expect(page).to have_content 'Register DOR Items'
       expect(page).to have_content 'Csv file missing headers: source_id.'
+
+      attach_file 'Upload a CSV file', file_fixture('item_registration.csv')
+
+      click_button 'Register'
+
+      expect(page).to have_content 'Register druids job was successfully created.'
+      expect(page).to have_content 'Bulk Actions'
+
+      expect(BulkAction).to have_received(:new).with(user:, action_type: 'RegisterDruidsJob')
+      expect(bulk_action).to have_received(:save)
+      expect(bulk_action).to have_received(:enqueue_job).with(
+        {
+          administrative_policy_object: non_default_apo.externalIdentifier,
+          content_type: 'https://cocina.sul.stanford.edu/models/object',
+          csv_file: "source_id,catkey,barcode,label\nfoo:123,,,My new object\n",
+          groups:
+            ["sunetid:#{user.login}",
+             'workgroup:sdr:administrator-role',
+             'workgroup:dlss:developers'],
+          initial_workflow: 'registrationWF',
+          project_name: 'X-Files',
+          rights_download: 'none',
+          rights_view: 'dark',
+          tags: ['i : believe', "Registered By : #{user.login}"]
```

Comment:
This makes sure that what the user entered in the form was retained even after an invalid CSV file was uploaded.